Main.java

```diff
@@ -9,7 +11,28 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Creates a scanner for user's input
+    Scanner input = new Scanner(System.in);
+
+    // Asks the users the size of the pizza
+    System.out.println("What is the size of the pizza (in cm):");
+
+    // Gets the size of the pizza from the user
+    int size = input.nextInt();
+
+    // Calculates and shows the subtotal, tax, and the final total for the user 
+    System.out.println("Subtotal: " + (0.75 + 0.99 + 0.50 * size));
+    System.out.println("Taxes: " + 0.13 * (0.75 + 0.99 + 0.50 * size));
+    System.out.println("Total: " + 1.13 * (0.75 + 0.99 + 0.50 * size));
```

Comment:
These numbers really should be in constants. Right now, these are "magic numbers" and mean almost nothing. Also, it is not a good idea to do all of your math inside of output statements. Instead, create a variable to store them in and output just the variable instead. It makes the code more readable!